docs/using/libraries/ruby/index.rst

```diff
@@ -140,43 +146,39 @@ Likewise, dictionaries constructed in Polar may be passed into Ruby methods.
 
 Enumerators
```

Comment:
This section (and the below comment) should probably talk about `Enumerable` instead of `Enumerator`. The changes in #485 (I think) make it such that you can enumerate over any enumerable (strings, lists, etc., in addition to explicit enumerators) with the `in` operator.

Reply:
bd8bea72